contracts/examples/ExampleReceiverDapp.sol

```diff
@@ -9,18 +9,23 @@ contract ExampleReceiverDapp is Application {
         bytes32 messageId, uint256 fromChainId, address fromDappAddress, address localLineAddress, bytes message
     );
 
-    address public msgLine;
+    // local line address
+    address public immutable LINE;
+    // remote dapp address
+    address public immutable DAPP;
 
-    constructor(address msgPort, address line) Application(msgPort) {
-        msgLine = line;
+    constructor(address line, address dapp) {
+        LINE = line;
```

Comment:
Only one line? What about a mapping? https://github.com/darwinia-network/darwinia-msgport/pull/98/files#diff-16734e70b01da4ec89a9940d9c489615f2aa5f03dface0389b299fdc0b0e95f8R15

Reply:
I think simple Dapp use one line is enough. 
You could add a `multiReceiverExampleDapp` for multi trusted line.
`multiReceiverExampleDapp` is more complex and may need to aggregate the multi messages.
https://github.com/MultiMessageAggregation/multibridge/blob/b633bae0a7afcb8552e8bf366e3341ae9389c4fe/src/libraries/EIP5164/ExecutorAware.sol#L34